projects/plots/plots/vizapp/app.py

```diff
@@ -39,7 +40,7 @@ def __init__(
         valid_frac: float,
         fftlength: float,
         device: str = "cpu",
-        vetos: bool = True,
+        vetos: Optional["VETO_CATEGORIES"] = None,
```

Comment:
Is `VETO_CATEGORIES` meant to be in quotes here?

Reply:
Yes youre right, I think I should remove the quotes and import it directly (instead of under the `if TYPE_CHECKING` guard)